packages/components/src/components/hds/dropdown/list-item/checkmark.ts

```diff
@@ -4,15 +4,29 @@
  */
 
 import Component from '@glimmer/component';
+import type { FlightIconSignature } from '@hashicorp/ember-flight-icons/components/flight-icon';
+import type { HdsInteractiveSignature } from '../../interactive';
 
-export default class HdsDropdownListItemCheckmarkComponent extends Component {
+export interface HdsDropdownListItemCheckmarkSignature {
+  Args: HdsInteractiveSignature['Args'] & {
+    count?: string | number;
```

Comment:
notice: in the documentation we say it's a `string` only. One of the two needs to be updated: https://helios.hashicorp.design/components/dropdown?tab=code#dcheckmark

Reply:
Updated in f30a60b2591d8c055ce35d711cc312a977cc4e8d for all `count` arguments